app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -400,8 +399,9 @@ class PatientSummaryScreenController @Inject constructor(
         .map { (appointmentDate, patient) -> Scheduled(patient.fullName, appointmentDate) as PatientSummaryResult }
 
     val wasPatientSummaryItemsChanged = events.ofType<PatientSummaryItemChanged>()
+        .map { it.patientSummaryItem }
+        .filter { it.hasItemChangedSince(Instant.now()) }
```

Comment:
Isn't this supposed to be the time passed from when the screen was created?

Reply:
This will be done in a different PR.